### DISCUSSION TITLE
How to merge two different indices with same type?

### DISCUSSION THREAD

**Discussion by lifelongeeek** [castorini/pyserini#1518] [Q&A]
How can I combine two different indices with same type?

* lucene example: `msmarco-v2-doc` and `wikipedia-kilt-doc`
* faiss example: `mrtydi-v1.1-arabic-mdpr-tied-pft-nq` and  `mrtydi-v1.1-bengali-mdpr-tied-pft-nq`

**Comment by lintool** [comment]
There's currently no code for doing this. Your options are to (a) open both indexes, query them, and merge the results, or (b) glom the corpora together and re-index.
